stdlib/influxdata/influxdb/to.go

```diff
@@ -1,15 +1,622 @@
 package influxdb
 
 import (
+	"context"
+	"fmt"
 	"github.com/influxdata/flux"
+	"github.com/influxdata/flux/codes"
+	"github.com/influxdata/flux/compiler"
+	"github.com/influxdata/flux/dependencies/influxdb"
+	"github.com/influxdata/flux/execute"
+	"github.com/influxdata/flux/internal/errors"
+	"github.com/influxdata/flux/interpreter"
+	"github.com/influxdata/flux/plan"
 	"github.com/influxdata/flux/runtime"
+	"github.com/influxdata/flux/semantic"
+	"github.com/influxdata/flux/stdlib/kafka"
+	"github.com/influxdata/flux/values"
+	lp "github.com/influxdata/line-protocol"
+	"github.com/opentracing/opentracing-go"
+	"sort"
+	"time"
 )
 
 // ToKind is the kind for the `to` flux function
 const ToKind = "to"
 
-var ToSignature = runtime.MustLookupBuiltinType("influxdata/influxdb", "to")
-
 func init() {
-	runtime.RegisterPackageValue("influxdata/influxdb", ToKind, flux.MustValue(flux.FunctionValueWithSideEffect(ToKind, nil, ToSignature)))
+	toSignature := runtime.MustLookupBuiltinType("influxdata/influxdb", "to")
+	runtime.RegisterPackageValue("influxdata/influxdb", ToKind, flux.MustValue(flux.FunctionValueWithSideEffect(ToKind, createToOpSpec, toSignature)))
+	flux.RegisterOpSpec(ToKind, func() flux.OperationSpec { return &ToOpSpec{} })
```

Comment:
You can remove this line. It doesn't do anything but remains in the codebase in a lot of places because it doesn't cause any harm and nobody has gone back to remove them.